### ISSUE TITLE
Upgrade to angular 7 to angular 8

### ISSUE THREAD

**Issue by eltonhp** [irossimoline/angular4-material-table#33]
Hello, 
I made the upgrade angular 7 to angular 8 on my project. 
And the error is occurring when the AOT compile at the production mode:
 
_ERROR in ..\angular4-material-table\src\app.module.ts(4,2): Error during template compile of 'AppModule'
  Function calls are not supported in decorators but 'ɵmakeDecorator' was called in 'NgModule'
    'NgModule' calls 'ɵmakeDecorator'._ 

Please, can you help?

**Comment by blavenie** [comment]
WIth Angular 8 i have another error : 
```
  Type '() => import("./node_modules/@angular/forms/forms").FormGroup' is not assignable to type '() => import("./node_modules/angular4-material-table/node_modules/@angular/forms/src/model").FormGroup'.
```
I think a simple recompilation with [this PR](https://github.com/irossimoline/angular4-material-table/pull/34) will fix this.

**Comment by irossimoline** [comment]
Merged and created v0.8.0 with it. Thank you @blavenie for your help!

**Comment by rsteaburdea** [comment]
Hello.

Is npm linked automatically to the latest version on github or should I do something different than `npm install --save angular4-material-table`?
